android/engine/src/main/java/org/smartregister/fhircore/engine/data/remote/model/response/ChildTreeNode.java

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
can we ask sever team to release a lib that holds these changes.. and we just add that lib as dependecy

Reply:
@rehammuzzamil is it possible do that?

Reply:
I believe it would be a good idea to build a separate repo/lib for the model classes of the practitioner details endpoint, which could be utilised by both the client and the server. This is only possible if we need an exact copy of classes on the client-side as well.
Please confirm, @RaaziaTarique .

Reply:
@RaaziaTarique  ChildTreeNode class is not used in the practitioner-details end-point. 
Please only refer to model classes inside the practitioner module. 
`practitioner/src/main/java/org/smartregister/extension/model`

Reply:
@rehammuzzamil yes I am using the exact same classes. Actually FhirPractitionerDetails class is using LocationHierarchy class and that class is indirectly using ChildTreeNode class

Reply:
Okay right, in this case, you will be needing all the model classes of the location module as well. @RaaziaTarique 
https://github.com/opensrp/hapi-fhir-opensrp-extensions/tree/main/location/src/main/java/org/smartregister/extension/model

Reply:
fhir-common lib is now released and used